recipe/meta.yaml

```diff
@@ -27,7 +28,7 @@ requirements:
     - setuptools
   run:
     - python
-    - applaunchservices >=0.3.0  # [osx]
+    - applaunchservices >=0.3.0  # [osx and (x86_64 or (arm64 and py >= 39))]
```

Comment:
This can stay as was since the `arm64 and py < 39` are skipped, i.e., everything after `osx` will be `true`.